store/internal/btreeadaptor.go

```
@@ -0,0 +1,68 @@
+// GetStoreType returns the type of the store.
+func (ts *BTreeStore[V]) GetStoreType() types.StoreType {
+	return types.StoreTypeDB
+}
+
+// CacheWrap branches the underlying store.
+func (ts *BTreeStore[V]) CacheWrap() types.CacheWrap {
+	return cachekv.NewGStore(ts, ts.isZero, ts.valueLen)
+}
+
+// CacheWrapWithTrace branches the underlying store.
+func (ts *BTreeStore[V]) CacheWrapWithTrace(w io.Writer, tc types.TraceContext) types.CacheWrap {
```

golangci-lint (1.64.8)

[warning] unused-parameter: parameter 'w' seems to be unused, consider removing or renaming it as _

(revive)